### ISSUE TITLE
Multiple Popup - second popup showing only on initial click 

### ISSUE THREAD

**Issue by Deepanghoshs** [MicrosoftEdge/WebView2Feedback#1881]
**Description**

we are having three html pages name Page1.html,Page2.html & Page3.html.
Loading the page1 to webview2 control

Page1.html  - Contains a link, on clicking the link on page1.html will load the page2.html as popup window and the JS inside the Pages2.html will call and show the popup window Page3.html

So by clicking the link in the page1.html will load the page2.html in webview2 window & Page3.html as popup in default browser

Issue:
On the new webview control with page1.html - if we click the link all the popups(page2 & page3) are shown. Once we close the popup and again click on the same link on the page1.html, it will show only the page2.html and page3 is not loading. So page3.html popup is working only once (on initial case only )

FYI : Same is working properly on default browser (Edge) 
**Version**
SDK: 1.0.1020.30
Runtime: 
Framework: WPF with .net framework 4.8
OS: Windows 10

**Comment by champnic** [comment]
Thanks for the bug report @Deepanghoshs! Are you handling the NewWindowRequested event in your app, or just letting the default popup behavior happen?

**Comment by Deepanghoshs** [comment]
@champnic not handling NewWindowRequested. We are using JavaScript to show the popups

**Comment by champnic** [comment]
@Deepanghoshs Thanks for confirming. Are you able to share the html/js for how you are opening the popups?

**Comment by Deepanghoshs** [comment]
@champnic thanks for the update.  We are abt to release our application. So we postponed this enhancement to the next version. For the time being we are using the default browser control.

**Comment by victorhuangwq** [comment]
Are you still facing this issue?
